CHANGELOG.md

```diff
@@ -81,18 +82,59 @@ Adding a new version? You'll need three changes:
  - [0.0.5](#005)
  - [0.0.4 and prior](#004-and-prior)
 
-## Unreleased
+## [3.1.0]
+
+> Release date: 2024-02-07
+
+### Highlights
+
+- 🔒 Kong Gateway's [secret vaults](kong-vault) now become a first-class citizen for Kubernetes users
+  thanks to the new `KongVault` CRD.
+- 🔒 Providing an Enterprise license to KIC-managed Kong Gateways becomes much easier thanks to the new `KongLicense` CRD
+  which is used to dynamically provision all the replicas with the latest license found in the cluster.
+- ✨ Populating a single field of `KongPlugin`'s configuration with use of a Kubernetes Secret becomes possible thanks 
+  to the new `KongPlugin`'s `configPatches` field.
+- 🔒 All sensitive information stored in the cluster is now sanitized while sending configuration to Konnect.
```

Comment:
We haven't done that until now but given that we now have this section it would make sense to consider linking some (all?) docs guides/references that discuss those topics.

Reply:
Yeah, that would make sense. I'll update them once all of the guides are merged.